avocado/core/nrunner/task.py

```diff
@@ -124,7 +124,7 @@ def are_dependencies_available(self, runners_registry=None):
         """
         if runners_registry is None:
             runners_registry = RUNNERS_REGISTRY_STANDALONE_EXECUTABLE
-        return self.runnable.pick_runner_command(runners_registry)
+        return self.runnable.pick_runner_command(self.runnable.kind, runners_registry)
```

Comment:
Nitpick: Even if a static method can be called either on the class or on an instance, because of this first argument, this is somehow odd.

An object's method should not be using as argument any of the object arguments. this is redundant. 

I would replace this with `Runnable.pick_runner_command()` like you are doing at lines 27, 62 and in your tests.